code/game/machinery/doors/airlock.dm

```diff
@@ -685,11 +685,29 @@ About the new airlock wires panel:
 				spark(src, 5)
 	return ..()
 
+#define ID_SCAN 1
+#define DISRUPT_MAIN_POWER 2
+#define DISRUPT_BACKUP_POWER 3
+#define DROP_DOOR_BOLTS 4
+#define UNELECTRIFY_DOOR 5
+#define CLOSE_DOOR 7 // Where did the 6 go ?
+#define DOOR_SAFETIES 8
+#define DOOR_SPEED 9
+#define BOLT_LIGHTS 10
+
+#define RAISE_DOOR_BOLTS 4
+#define ELECTRIFY_30SEC 5
+#define ELECTRIFY_4EVER 6
+#define OPEN_DOOR 7
+
 /obj/machinery/door/airlock/Topic(href, href_list, var/nowindow = 0)
 	// If you add an if(..()) check you must first remove the var/nowindow parameter.
 	// Otherwise it will runtime with this kind of error: null.Topic()
 	var/turf/T = get_turf(usr)
-	if(!isAI(usr) && T.z != z)
+
+	var/user_is_AI = isAI(usr)
+
+	if(!user_is_AI && T.z != z)
```

Comment:
>this whole block

what am I missing here?

Reply:
Remote signalers do not work accross z-levels